charts/dapr/crds/httpendpoints.yaml

```diff
@@ -0,0 +1,102 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.5.0
+  creationTimestamp: null
+  name: httpendpoints.dapr.io
+spec:
+  group: dapr.io
+  names:
+    kind: HTTPEndpoint
+    listKind: HTTPEndpointList
+    plural: httpendpoints
+    singular: httpendpoint
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: HTTPEndpoint describes an Dapr HTTPEndpoint type to allow for external service invocation. This can include being external to Dapr, as well as being external to the environment.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          auth:
+            description: Auth represents authentication details for the component.
+            properties:
+              secretStore:
+                type: string
+            required:
+            - secretStore
+            type: object
+          kind:
+            description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          scopes:
+            items:
+              type: string
+            type: array
+          spec:
+            description: HTTPEndpointSpec describes an access specification for allowing potentially external service invocations.
+            properties:
+              allowed:
+                items:
+                  description: APISpec describes the configuration for Dapr API communication with potentially external services.
+                  properties:
+                    baseUrl:
+                      type: string
+                    headers:
+                      additionalProperties:
+                        type: string
+                      type: object
+                    name:
+                      type: string
+                    protocol:
+                      type: string
+                  required:
+                  - baseUrl
+                  - name
+                  type: object
+                type: array
+              metadata:
+                items:
+                  description: MetadataItem is a name/value pair for a metadata.
+                  properties:
+                    name:
+                      type: string
+                    secretKeyRef:
```

Comment:
can secretKeyRef be part of headers? 
As in if a basic auth header is needed, how is that going to be specified in the `headers` field?

Reply:
In the [updated proposal ](https://github.com/dapr/proposals/pull/25)I moved `secretKeyRef` under the headers section to this is more easily done.